index.html

```diff
@@ -0,0 +1,185 @@
+        <li class="mobile-menu-item">
+          <img id="cancel-btn" src="assets/img/cancelBtn.png" alt="Cancel">
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html">About</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#program-section">Features</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#advisors-section">Advisors</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html#partners-section">Partners</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="https://www.dubaiairshow.aero/news/industry-news" target="_blank" rel="noopener noreferrer">Media</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="nav-bar">
+      <nav id="nav-bar1">
+        <div class="social">
+          <a><i class="fab fa-facebook-f"></i></a>
+          <a><i class="fab fa-twitter"></i></a>
+        </div>
+        <ul class="nav-bar1-list">
+          <li class="nav-bar1-item">
+            <a href="#">Arabic</a>
+          </li>
+          <li class="nav-bar1-item">
+            <a href="#">Register</a>
+          </li>
+          <li class="nav-bar1-item">
+            <a href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="nav-bar2">
+        <a id="logo" href="index.html"></a>
+        <ul class="nav-bar2-list">
+          <li class="nav-bar2-item">
+            <a href="about.html">About</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#program-section">Features</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#advisors-section">Advisors</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="#partners-section">Partners</a>
+          </li>
+          <li class="nav-bar2-item">
+            <a href="https://www.dubaiairshow.aero/news/industry-news" target="_blank" rel="noopener noreferrer">Media</a>
+          </li>
+          <li id="exhibition" class="nav-bar2-item">
+            <a href="https://www.dubaiairshow.aero/floor-plan-download" target="_blank" rel="noopener noreferrer">Exhibition</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section id="headline-section">
+    <h3 class="subtitle">The future of aerospace industry</h3>
+    <div id="mask">
+      <h1 id="headline-title">
+        DUBAI AIRSHOW
+        <br>
+        EXHIBITION
+        <br>
+        2023
+      </h1>
+    </div>
+    <p class="description">
+      A biennial air show held in Dubai and organised by Tarsus Aerospace since 1989, 
+      and it is open to business professionals and industry only.
+    </p>
+    <h3 class="schedule">2023.11.13(MON) ~ 17(FRI)</h3>
+    <p id="location">@ Dubai World Central, United Arab Emirates</p>
+  </section>
+  <section id="program-section">
+    <h2 class="program-heading">Show Features
+      <hr class="divider">
```

Comment:
- [ ]  You design looks good, but this part should be improved to resemble it to the provided design.  Your page have a very bigger space left because the font styles used are smaller. please increase the size so that it can look much closer to the design 

Your design: 
![image](https://github.com/tajemouti/first-capstone/assets/43877689/14d88bfc-37fd-4e55-8d35-b10782070e9d)

Provided design: 
![image](https://github.com/tajemouti/first-capstone/assets/43877689/b130706e-e45c-4c61-bfdc-c30572450df3)